solution_template/jenkins/nested/VM-password.json

```diff
@@ -0,0 +1,147 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "outputs": {
+  },
+  "parameters": {
+    "adminPassword": {
+      "metadata": {
```

Comment:
I don't really see the point in having a description for nested template parameters. Seems easier to just remove them rather than maintain them